story.txt

```diff
@@ -1,3 +1,4 @@
 I was on my way to the Baeckerei
 When I saw an ad that Rammstein was going to be performing at the olympiastadion in Berlin this summer.
-So I went over to the Haupstrasse and started heading towards the Flughafen
```

Comment:
the 3rd sentence looks like it got changed due to difference in line endings (linux vs windows). But that doesn't make any noticeable different to the story. LGTM